Audiobook Player/PlayerManager.swift

```diff
@@ -372,7 +378,9 @@ extension PlayerManager: AVAudioPlayerDelegate {
         if storedPercentage != percentageString {
             defaults.set(percentageString, forKey: self.identifier+"_percentage")
 
-            NotificationCenter.default.post(name: Notification.Name.AudiobookPlayer.updatePercentage, object: nil, userInfo: userInfo)
+            DispatchQueue.main.async {
```

Comment:
`updateTimer` should already be on the main thread due to the timer being added to the `RunLoop.main`. I did some testing inserting `if Thread.isMainThread { print("on main thread") }` and it prints on every tick. Let me know in which specific case these notifications have been going on a background thread